playbooks/common/openshift-master/config.yml

```diff
@@ -180,6 +180,7 @@
     openshift_master_hosts: "{{ groups.oo_masters_to_config }}"
   - role: nickhammond.logrotate
   - role: nuage_master
+    serial: 1
```

Comment:
@detiber You are correct. I basically need to somehow serialize the following task on the node

https://github.com/openshift/openshift-ansible/blob/master/roles/nuage_node/tasks/certificates.yml#L16

The issue is the serial file on the master is getting updated (and corrupted) by multiple node tasks running in parallel